### PR TITLE
Updated infobox_templates

### PR DESCRIPTION
**Pull request by Sharingan-Coder01**
This PR is to add the infobox-template for the nlwiki article quality model. It is an update to the previous PR where the small changes made were missing.

### CODE REVIEW

File: articlequality/feature_lists/nlwiki.py [hunk 1]
```diff
@@ -54,6 +54,8 @@
         max(wikitext.revision.content_chars, 1),
     cn_templates,
     cn_templates / max(wikitext.revision.content_chars, 1),
+    infobox_templates,
+    infobox_templates / max(wikitext.revision.content_chars, 1),
```
Comment (halfak) [inline articlequality/feature_lists/nlwiki.py:58]: This second feature is probably not necessary but no real harm in it.

File: model_info/nlwiki.wp10.md [hunk 2]
```diff
@@ -34,51 +34,51 @@ Model Information:
...
+		0.922  0.949  0.967  0.909  0.924
+	f1 (micro=0.738, macro=0.738):
+		    E      D      C      B    A
+		-----  -----  -----  -----  ---
+		0.793  0.692  0.867  0.636  0.7
+	!f1 (micro=0.933, macro=0.933):
+		    E      D      C     B      A
+		-----  -----  -----  ----  -----
+		0.959  0.905  0.967  0.91  0.926
+	accuracy (micro=0.894, macro=0.894):
```
Comment (halfak) [inline model_info/nlwiki.wp10.md:66]: Looks like we lost a bit of accuracy.  That could be related to random chance.   The difference is small.

Comment (Sharingan-Coder01) [reply model_info/nlwiki.wp10.md:66]: Yes I thought so too! Do you think this changes are required?

File: articlequality/feature_lists/nlwiki.py (hunk 1 above)
Comment (Sharingan-Coder01) [reply articlequality/feature_lists/nlwiki.py:58]: Alright!

**Comment by elukey** [comment]
Hi folks, thanks a lot for this work. If possible I'd ask a couple of things:

1) a description of what has been done in the pull request description msg (it is empty at the moment).
2) a more meaningful set of commit messages explaining briefly what has been done and why (some of them do have typos too).

**Comment by Sharingan-Coder01** [comment]
> Hi folks, thanks a lot for this work. If possible I'd ask a couple of things:
> 
>     1. a description of what has been done in the pull request description msg (it is empty at the moment).
> 
>     2. a more meaningful set of commit messages explaining briefly what has been done and why (some of them do have typos too).

Hi! 
Sorry for that, it was a very minor change and I must've not rechecked properly. Will make sure this does not happen in the future. I have added a short description too.

**Pull request closed without merging**